### PR TITLE
fix(core/webidl): autolink worker types

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

File: src/core/webidl.js [hunk 1]
```diff
@@ -38,9 +38,11 @@ const templates = {
       : // Other keywords like sequence, maplike, etc...
         hyperHTML`<a data-xref-type="dfn" data-cite="WebIDL">${keyword}</a>`;
   },
-  reference(wrapped, unescaped) {
+  reference(wrapped, unescaped, context = {}) {
```
Comment (saschanaz) [inline src/core/webidl.js]: Oh, do we get undefined context sometimes? That's unexpected.

File: src/core/webidl.js [hunk 2]
```diff
@@ -49,9 +51,22 @@ const templates = {
       case "object":
         type = "interface";
         cite = "WebIDL";
+        break;
+      default: {
+        const isWorkerType = unescaped.includes("Worker");
```
Comment (saschanaz) [inline src/core/webidl.js]: We also have worklets: https://webaudio.github.io/web-audio-api/#audioworkletglobalscope

File: src/core/webidl.js (hunk 1 above)
Comment (marcoscaceres) [reply src/core/webidl.js]: No, that was my silliness from before I realized the WebIDL2.js hadn’t updated.

File: src/core/webidl.js [hunk 3]
```diff
@@ -49,9 +51,22 @@ const templates = {
...
+        if (isWorkerType && tokens && tokens.name.value === "Exposed") {
```
Comment (saschanaz) [inline src/core/webidl.js]:
```suggestion
        if (isWorkerType && context.type === "extended-attribute" && context.name === "Exposed") {
```

File: src/core/webidl.js [hunk 4]
```diff
@@ -49,9 +51,22 @@ const templates = {
...
+          lt = `${unescaped}GlobalScope`;
+          type = "interface";
+          cite = ["Worker", "DedicatedWorker", "SharedWorker"].includes(
```
Comment (saschanaz) [inline src/core/webidl.js]: Is this some optimization thing?

Comment (marcoscaceres) [reply src/core/webidl.js]: No, cite needs to be explicit for now or it breaks.

File: src/core/webidl.js (hunk 2 above)
Comment (marcoscaceres) [reply src/core/webidl.js]: Let’s add these in the next round.

**Comment by marcoscaceres** [comment]
Travis got stuck...

**Comment by saschanaz** [comment]
Looks like we have both Travis app and service, maybe we can remove service now.

**Comment by marcoscaceres** [comment]
Done!

**Pull request merged**